record_imu_cam/README.md

```diff
@@ -0,0 +1,62 @@
+# Record IMU Cam
+
+The purpose of this plugin is to record a dataset, similar to the [EuRoC MAV dataset][1] that includes the IMU data and Cam images.
+    
+
+## How to record a dataset
+
+In `configs/rt_slam_plugins.yaml`, uncomment this line:
+
+    - path: record_imu_cam/
+
+After recording, the dataset will be stored in the ILLIXR project directory, with the following structure:
+
+    ILLIXR/data_record
+        \_ cam0/
+            \_ data/ 
+                \_ timestamp.png
+                \_ ...
+            \_ data.csv
+        \_ cam1/
+            \_ data/
+                \_ timestamp.png
+                \_ ...
+            \_ data.csv
+        \_ imu0
+            \_ data.csv
+
+### Format
+
+1. `cam0/data.csv` and `cam1/data.csv` are both formatted as 
+   
+        timestamp [ns], timestamp.png
+2. `imu0/data.csv` is formatted as 
+    
+        timestamp [ns],w_x [rad s^-1],w_y [rad s^-1],w_z [rad s^-1],a_x [m s^-2],a_y [m s^-2],a_z [m s^-2]
+
+## How to rerun recorded dataset
+
+1. **(IMPORTANT)** In `configs/rt_slam_plugins.yaml`, comment this line: 
+
+       # - path: record_imu_cam/  
+
+2. In `configs/native.yaml` (or whatever mode you're running ILLIXR with), add the path of the recorded dataset like so:
```

Comment:
I don't know if this is how we should use data variable as  input and output variable path.

Reply:
That part is for running the recorded dataset that is captured from the `record_imu_cam` plugin. The default output path is `ILLIXR/`